spark_log_parser/loaders/https.py

```diff
@@ -25,8 +28,8 @@ def load_item(self, url):
         if not int(response.headers.get("Content-Length", 0)):
             raise AssertionError("Download is empty")
 
-        stream = response.raw
-        yield from self.extract(Path(url), stream)
+        wrapped = FileChunkStreamWrapper(response.iter_content(chunk_size=self._STREAM_CHUNK_SIZE))
+        yield from self.extract(Path(url), wrapped)
```

Comment:
Wrap our chunk iterator in our "file-like" object. We want to force these streams to chunk when we read from them, whereas right now, modules like e.g. `gzip` may issue single-byte reads to the underlying response stream, which isn't ideal